pkg/crc/machine/machine.go

```diff
@@ -253,17 +255,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 	// Check the certs validity inside the vm
 	logging.Info("Verifying validity of the cluster certificates ...")
-	expiringIn7Days, duration, err := cluster.CheckCertsValidity(sshRunner)
+	err = checkVMCertsExpiry(sshRunner)
```

Comment:
```
valid, err = checkVMCertsExpiry(sshRunner)

if !valid && err != nil {
  
```

as this would make the code easier to understand than having to interfere the use of an error to communicate this state

Reply:
Does 'valid' mean 'certificate is expiring in more than 7 days'? Or something else?

Reply:
enum option is better, as this tries to capture a tri-state; valid, still valid and expired.